src/_settings/mappings/career.json

```diff
@@ -0,0 +1,133 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "applicant_types": {
+      "properties": {
+        "announcement_number": {
+          "type": "string"
+        },
+        "application_type": {
+          "properties": {
+            "active": {
+              "type": "boolean"
+            },
+            "applicant_type": {
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
+            },
+            "id": {
+              "type": "long"
+            },
+            "slug": {
+              "type": "string"
+            }
+          }
+        },
+        "id": {
+          "type": "long"
+        },
+        "is_usajobs": {
```

Comment:
Are there postings that aren't USAJobs? We'll need to swap out some content in the page if so.